src/cpp/src/tokenizer.cpp

```diff
@@ -137,15 +130,31 @@ class Tokenizer::TokenizerImpl {
         ov::Tensor skip_special_tensor = ov::Tensor(ov::element::i32, {1});
         *skip_special_tensor.data<int>() = skip_special_tokens_flag;
 
+        ov::Tensor max_trunc_length_tensor = ov::Tensor(ov::element::i32, {});
+        *max_trunc_length_tensor.data<int>() = max_trunc_length_val;
+        ov::Tensor max_pad_length_tensor = ov::Tensor(ov::element::i32, {1});
+        *max_pad_length_tensor.data<int>() = max_pad_length_val;
```

Comment:
ints aren't always 32 bit. Use int32_t. Although if someone tries to compile genai on such system it would probably fail earlier. You use int32_t in `get_padding_values()` after all

Reply:
done